src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -89,9 +89,10 @@ public static List<GeneratableElement> getModElementExamplesFor(Workspace worksp
 			generatableElements.add(getToolExample(me(workspace, type, "10"), "MultiTool", random, true, false));
 			generatableElements.add(getToolExample(me(workspace, type, "11"), "Shears", random, true, false));
 			generatableElements.add(getToolExample(me(workspace, type, "12"), "Fishing rod", random, true, false));
-		} else if (type == ModElementType.FUEL || type == ModElementType.TAB) {
+		} else if (type == ModElementType.ITEMEXTENSION || type == ModElementType.TAB) {

```

Comment:
This section that adss only 3 tests is for really simple elements that ITEMEXTENSIO, so move ITEMEXTENSION to standard tests